src/js/index.js

```diff
@@ -60,15 +60,19 @@ function handleSort() {
 
 function handleTags() {
 	const tags = selectr.getValue();
+	const cards = Array.from(contentElement.querySelectorAll('.item'));
 
-	Array.from(contentElement.querySelectorAll('.item')).forEach(element => {
+	cards.forEach(element => {
 		const show = (showExpired.checked || !element.classList.contains('tag-expired')) &&
 			tags.reduce((sho, tag) => sho || element.classList.contains(`tag-${tag}`), tags.length === 0);
 		if (!show) {
 			element.classList.remove('visible');
 		}
 	});
 
+	const hasResults = cards.every(element => !element.classList.contains('visible'));
```

Comment:
Rather than doing a double array loop, I'd rather just compute this in the forEach loop - e.g.

```javascript
let hasResults = false;

cards.forEach(element => {
	const show = // ...

	hasResults = hasResults || show;
});
```